src/hooks/offramp/useOfframpEvents.ts

```diff
@@ -0,0 +1,45 @@
+import { useCallback, useState } from 'preact/compat';
+
+import { EventStatus } from '../../components/GenericEvent';
+import { GenericEvent } from '../../components/GenericEvent';
+
+import { createTransactionEvent } from '../../contexts/events';
+import { useEventsContext } from '../../contexts/events';
+import { useNetwork } from '../../contexts/network';
+
+import { getInputTokenDetailsOrDefault } from '../../constants/tokenConfig';
+import { OfframpingState } from '../../services/offrampingFlow';
+import { OFFRAMPING_PHASE_SECONDS } from '../../pages/progress';
+
+export const useOfframpEvents = () => {
+  const { trackEvent, resetUniqueEvents } = useEventsContext();
+  const { selectedNetwork } = useNetwork();
+  const [_, setEvents] = useState<GenericEvent[]>([]);
+
+  // @todo: why do we need this?
+  const addEvent = (message: string, status: EventStatus) => {
```

Comment:
Not at all! this comes from way way before where it used to display the logs on the UI. We've been dragging parts of it for a while now apparently.

Reply:
@gianfra-t Can I remove it?

Reply:
Yes sure! I hope is not so much work